botcore/_bot.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         self._resolver: Optional[aiohttp.AsyncResolver] = None
         self._connector: Optional[aiohttp.TCPConnector] = None
 
-        self.statsd_url: Optional[str] = None
+        self.statsd_url: Optional[str] = kwargs.get("statsd_url", None)
```

Comment:
Since it is a custom kwarg (i.e. not inherited from discord.py's `Bot`), it should be documented.

Reply:
Force pushed a change to make this a named arg, with documentation.